src/includes/enriching-events/attach-screenshots/unity.mdx

```diff
@@ -0,0 +1,13 @@
+The **Attach Screenshot** options can be found on the `Enrichment` tab in the Editor Configuration Window.
+
+![Screenshots Thumbnail Example](unity-screenshots.png)
+
+<Note>
+
+Note that capturing a screenshot in the Unity SDK requires running on the main thread.
+
+</Note>
+
+### Max Width and Max Height
+
+The default value of 0 means that screenshots get captured in the game's current [Screen](https://docs.unity3d.com/ScriptReference/Screen.html) size. With max values greater than 0 but less than the screen size, the SDK will try to scale down the captured screenshot while preserving the apect ratio. This allows for a smaller file being sent.
```

Comment:
I've made a change to the last sentence that I'm hoping doesn't change the meaning too much. If it does, lemme know and I'll try to edit a different way.
```suggestion
The default value of 0 means that screenshots are captured in the game's current [screen](https://docs.unity3d.com/ScriptReference/Screen.html) size. If max values are greater than 0 but less than the screen size, the SDK will try to scale down the captured screenshot while preserving the aspect ratio. This allows for a smaller file to be generated.
```